Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/azuredeploy.json

```diff
@@ -0,0 +1,320 @@
+                ],
+                "properties": {
+                    "CohesityQueueName": "[variables('QueueName')]",
+                    "keyVaultName": "[variables('ProducerFunctionName')]",
+                    "Workspace": "[parameters('Workspace')]",
+                    "AzureWebJobsStorage": "[concat('DefaultEndpointsProtocol=https;AccountName=', toLower(variables('ProducerFunctionName')),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', toLower(variables('ProducerFunctionName'))), '2019-06-01').keys[0].value, ';EndpointSuffix=',toLower(variables('StorageSuffix')))]",
+                    "BlobStorageConnectionString": "[concat('DefaultEndpointsProtocol=https;AccountName=', toLower(variables('ProducerFunctionName')),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', toLower(variables('ProducerFunctionName'))), '2019-06-01').keys[0].value, ';EndpointSuffix=',toLower(variables('StorageSuffix')))]",
+                    "startDaysAgo": "-30"
+                }
+            }]
+        },
+        {
+            "type": "Microsoft.Web/sites",
+            "apiVersion": "2018-11-01",
+            "name": "[variables('ConsumerFunctionName')]",
+            "location": "[resourceGroup().location]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Storage/storageAccounts', tolower(variables('ProducerFunctionName')))]"
+            ],
+            "kind": "functionapp",
+            "identity": {
+                "type": "SystemAssigned"
+            },
+            "properties": {
+                "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', variables('hostingPlanName'))]",
+                "name": "[variables('ConsumerFunctionName')]",
+                "httpsOnly": true,
+                "clientAffinityEnabled": true,
+                "alwaysOn": true,
+                "reserved": true
+            },
+            "resources": [{
+                "type": "config",
+                "apiVersion": "2018-11-01",
+                "name": "appsettings",
+                "dependsOn": [
+                    "[concat('Microsoft.Web/sites/', variables('ConsumerFunctionName'))]"
+                ],
+                "properties": {
+                    "CohesityQueueName": "[variables('QueueName')]",
+                    "keyVaultName": "[variables('ProducerFunctionName')]",
+                    "resourceGroup": "[resourceGroup().Name]",
+                    "subscription": "[subscription().subscriptionId]",
+                    "TenantId": "[subscription().tenantId]",
+                    "AzureWebJobsStorage": "[concat('DefaultEndpointsProtocol=https;AccountName=', toLower(variables('ProducerFunctionName')),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', toLower(variables('ProducerFunctionName'))), '2019-06-01').keys[0].value, ';EndpointSuffix=',toLower(variables('StorageSuffix')))]",
```

Comment:
Has AzureWebJobStorage has the same value as BlobStorageConnection string?
If it has, then why have 2 and not just use one variable?

Reply:
done in the next push.